docs/vsphere-prerequisites.md

```diff
@@ -1,14 +1,19 @@
 # vSphere pre-requisites
 
+The vSphere cluster has to be configured with [hybrid OVN Kubernetes networking](docs/setup-hybrid-OVNKubernetes-cluster.md)
+with a custom VXLAN port to work around the pod-to-pod connectivity between
+hosts [issue](https://docs.microsoft.com/en-us/virtualization/windowscontainers/kubernetes/common-problems#pod-to-pod-connectivity-between-hosts-is-broken-on-my-kubernetes-cluster-running-on-vsphere).
+
 In order to use the Windows MachineConfig Operator to successfully create
 a Windows node, you will first need to create a Windows "golden image"
 VM. These steps are an overview of the required configuration of the
 golden image.
 
 ## vSphere Windows VM golden image creation
 * Create the VM from an updated version of Windows Server 1909 VM image
-which includes patch
-[KB4565351](https://support.microsoft.com/en-us/help/4565351/windows-10-update-kb4565351)
+  that includes the patch [KB4565351](https://support.microsoft.com/en-us/help/4565351/windows-10-update-kb4565351).
+  This is required to get the OS feature that allows setting the VXLAN UDP port.
+  Please note that this patch is not available for `Windows Server 2019`.
```

Comment:
So this is a bit confusing. Are we saying...

* User Windows Server 2019 with this patch
* This patch isn't available for Windows Server 2019

...if so then there's no way for the user to be in a supported state :)

If we mean something different we should clarify

Reply:
I am saying using **1909** :-)

Reply:
Nevermind. I see now you're talking about 1909.

Reply:
LGTM